tests/zenko_e2e/backbeat/Using.md

```diff
@@ -47,10 +57,10 @@ export AWS_SOURCE_LOCATION=<source-location-name>
 3. Export the access key, secret key, AWS bucket name, and AWS location:
```

Comment:
For consistency, I would suggest adding the `source` instructions in this section as well. I agree with you, it would help ensure that it's not forgotten.

Reply:
Thanks! Updated.